doc/content/toolstack/features/HA/index.md

```diff
@@ -551,10 +551,10 @@ sig
 end
 ```
 
-The function `get_specific_plan` takes a configuration and a list of Hosts
-which have failed. It returns a VM restart plan represented as a VM to Host
-association list. This is the function called by the
-background HA VM restart thread on the master.
+The function `get_specific_plan` takes a configuration and a list of VMs,
+the host where they are resident have failed. It returns a VM restart plan
```

Comment:
Do you mean a list of `(VM, host)` pair where the host is the one on which VM is resident on?

Reply:
Sorry, I didn't make it clear, I mean just a list of `VM`. The hosts where those VMs are on have failed.
Updated to:

`... a list of VMs(the host where they are resident on have failed).
`